backend/internal/controller/server/server.go

```diff
@@ -2,24 +2,32 @@ package server
 
 import (
 	"net/http"
-	"remote/internal/controller/server/handlers"
+
+	"github.com/HackYourCareer/SmartKickers/internal/model"
 
 	"github.com/gorilla/mux"
 )
 
+type Server interface {
+	Start() error
+}
 type server struct {
 	router  *mux.Router
 	address string
+	game    model.Game
```

Comment:
This should be an interface, passed through the server's constructor

Reply:
Changed